samples/bot-formatting-cards/csharp/BotFormattingCards/Cards/AllCards.cs

```diff
@@ -151,5 +151,43 @@ public static Attachment sendCardWithEmoji()
             return emojiAdaptiveCardAttachment;
         }
 
+        /// <summary>
+        /// Persona card Icon in an Adaptive Card
+        /// If you want to show a single user in an Adaptive Card, the Adaptive Card displays the people icon and the name of the user.
+        /// </summary>
+        /// <returns>Return Microsoft.Bot.Schema.Attachment results.</returns>
+        public static Attachment sendPersonaCardIcon()
+        {
+            var paths = new[] { ".", "Resources", "adaptivePeoplePersonaCardIcon.json" };
+            var adaptiveCardPersonaCardIconJson = File.ReadAllText(Path.Combine(paths));
+
+            var PersonaCardIconAdaptiveCardAttachment = new Attachment()
+            {
+                ContentType = "application/vnd.microsoft.card.adaptive",
```

Comment:
If we are using it at multiple places, please create a constant in same file.

Reply:
Fixed